### ISSUE TITLE
Vault disabled but enabled at the same time?

### ISSUE THREAD

**Issue by theprodigy19444** [MilkBowl/Vault#835]
Upon starting up my server, I get the following error in my console:
07.12 14:56:28 [Server] Server thread/INFO Enabling EconomyShopGUI v2.4.3
07.12 14:56:28 [Server] Server thread/INFO Using lang-en.yml as language file.
07.12 14:56:28 [Server] Server thread/WARN [ERROR] Disabled due to no Vault dependency found!
07.12 14:56:28 [Server] Server thread/INFO Disabling EconomyShopGUI v2.4.3

Then when enabling CrazyEnchantments, it pops up this in the console:
07.12 14:56:28 [Server] Server thread/INFO Crazy Enchantment Hooks
07.12 14:56:28 [Server] Server thread/INFO VAULT: Enabled
07.12 14:56:28 [Server] Server thread/INFO MCMMO: Enabled
07.12 14:56:28 [Server] Thread-13/WARN The NBT-API at 'package me.badbones69.crazyenchantments.nbtapi' seems to be outdated!
07.12 14:56:28 [Server] Thread-13/WARN Current Version: '2.5.0' Newest Version: 2.6.0'
07.12 14:56:28 [Server] Thread-13/WARN Please update the nbt-api or the plugin that contains the api!

Current plugins added:
Vault, EconomyShopGUI, EssentialsX, AdvancedTeleport, One Player Sleep, EmeraldTools, CrazyEnchantments, mcMMO and DeluxeMenus.

No other errors are found in console except for DeluxeMenus and it doesn’t even show up in my plugins folder, but I’ll take that issue to their issue system.

Looking at it, it’s saying Vault is disabled but also enabled at the same time? I have all the most recent versions of the plugins except for CrazyEnchantments api but that wouldn’t effect Vault would it?

Any help would be greatly appreciated. Thank you!

**Comment by theprodigy19444** [comment]
Update:
Fixed the DeluxeMenus issue, didn’t have PlaceHolderAPI installed, installed it and that is fixed. 
I did notice this earlier in the console, not sure if it helps identifying the Vault issue tho:
07.12 15:30:44 [Server] Server thread/WARN Loaded class net.milkbowl.vault.permission.Permission from Vault v1.7.3-b131 which is not a depend, softdepend or loadbefore of this plugin.
07.12 15:30:44 [Server] Server thread/INFO Registered Vault permission & chat hook.
